src/components/tipRecords/TipRecord.vue

```diff
@@ -206,18 +210,21 @@ export default {
       return isPinned !== -1;
     },
   },
+  mounted() {
+    if (this.$route.name !== 'tip') {
```

Comment:
it disables callback handler, but pin button will be visible on all pages, will it work in case if web wallet used on not 'tip' page? I'm not sure that we need this restriction.

Reply:
Yes, it works, the action redirects to the tip specific page upon wallet redirect. That's why the url(param) for the success callback is passed. Otherwise every tip component on the page would execute the callback.